pkg/operator/clustermembercontroller/clustermembercontroller.go

```diff
@@ -270,6 +278,20 @@ func (c *ClusterMemberController) sync() error {
 			return updateError
 		}
 		klog.Infof("All cluster members observed, scaling complete!")
+		if c.IsMember("etcd-bootstrap") {
+			c.eventRecorder.Event("BootstrapSafeToRemove", "scaling complete, bootstrap safe to remove")
+			_, _, updateErr := v1helpers.UpdateStatus(c.operatorConfigClient, v1helpers.UpdateConditionFn(
+				operatorv1.OperatorCondition{
+					Type:    ConditionBootstrapSafeToRemove,
+					Status:  operatorv1.ConditionTrue,
+					Reason:  "ScalingComplete",
+					Message: "cluster-etcd-operator has scaled, bootstrap safe to remove",
+				}))
+			if updateErr != nil {
+				klog.Errorf("clustermembercontroller:sync: error updating status: %#v", updateErr)
+				return updateErr
+			}
+		}
```

Comment:
else, set this condition to false. You need to write a level driven controller which observes the actual state and writes the correct conditions, not an edge triggered one that simply sets conditions once, one way.

Any time you add a condition, **always** ensure that you have code that sets it both true and false. If you don't, you wrote a bug.

Reply:
this is still outstanding